quickstart/.env

```diff
@@ -90,9 +175,10 @@ SCRIBE_POSTGRES_HOST=${SPLICE_APP_PROVIDER_DB_SERVER}
 SCRIBE_POSTGRES_USER=${SPLICE_APP_PROVIDER_DB_USER}
 SCRIBE_POSTGRES_PASSWORD=${SPLICE_APP_PROVIDER_DB_PASSWORD}
 SCRIBE_POSTGRES_DATABASE=scribe
-SCRIBE_PIPELINE_OAUTH_CLIENTID=${AUTH_APP_PROVIDER_CLIENT_ID}
-SCRIBE_PIPELINE_OAUTH_CLIENTSECRET=${AUTH_APP_PROVIDER_CLIENT_SECRET}
-SCRIBE_PIPELINE_OAUTH_ENDPOINT=${AUTH_APP_PROVIDER_TOKEN_URI}
+SCRIBE_PIPELINE_OAUTH_CLIENTID=${AUTH_APP_PROVIDER_PQS_CLIENT_ID}
+SCRIBE_PIPELINE_OAUTH_CLIENTSECRET=${AUTH_APP_PROVIDER_PQS_CLIENT_SECRET}
+SCRIBE_PIPELINE_OAUTH_SCOPE=openid
+SCRIBE_PIPELINE_OAUTH_ENDPOINT=${AUTH_APP_PROVIDER_TOKEN_URL}
```

Comment:
Not on this PR, but on another one: consider using `SCRIBE_APP_PROVIDER` as the prefix to make it clear who runs this.

Reply:
It would also be worth referring to `SCRIBE` as `PQS` as this is the name we are focusing on 👍🏻